recipes/googlemaps/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "googlemaps" %}
+{% set version = "2.5.1" %}
+{% set sha256 = "fcd5792aa5dae3c5d7200da64dd6494ff0a1b9ac4d1ff50c0b4f7ad788a3f10f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - requests >=2.11.1,<3.0
+    - setuptools
+  run:
+    - python
+    - requests >=2.11.1,<3.0
+
+test:
+  imports:
+    - googlemaps
+
+about:
+  home: http://github.com/simplejson/simplejson
+  license: Apache 2
+  license_family: Apache
```

Comment:
Please manually package the license and add an issue upstream:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually